pkg/scannerv4/mappers/mappers_test.go

```diff
@@ -899,6 +899,223 @@ func Test_toProtoV4Contents(t *testing.T) {
 	}
 }
 
+func Test_toProtoV4VulnerabilitiesMapWithEPSS(t *testing.T) {
+	now := time.Now()
+	protoNow, err := protocompat.ConvertTimeToTimestampOrError(now)
+	assert.NoError(t, err)
+
+	tests := map[string]struct {
+		ccVulnerabilities map[string]*claircore.Vulnerability
+		nvdVulns          map[string]map[string]*nvdschema.CVEAPIJSON20CVEItem
+		pkgEpss           map[string]epssDetail
+		enableRedHatCVEs  bool
+		want              map[string]*v4.VulnerabilityReport_Vulnerability
+	}{
+		"EPSS OK with NVD": {
+			ccVulnerabilities: map[string]*claircore.Vulnerability{
+				"foo": {
+					ID:      "foo",
+					Name:    "Name contains CVE-1234-567",
+					Issued:  now,
+					Updater: "unknown updater",
```

Comment:
Why did you have to setup the updater?

Reply:
CVSS Metrics method checks for updater names